plans_console.py

```diff
@@ -820,12 +884,20 @@ def doOperClicked(self):
         selFeatureTitle=self.ui.selFeature.text()
         ## check that the shapes exist
         selFeature=self.sts.getFeature(title=selFeatureTitle,featureClassExcludeList=['Folder','OperationalPeriod'])
+        if selFeature == -1:
+            logging.warning(op+' operation failed: Selected feature "'+selFeatureTitle+'" has an issue.')
+            inform_user_about_issue(op+' operation failed:\n\nThere are more than one feature with the name "'+selFeatureTitle+'"')
+            return
         if not selFeature:
-            logging.warning(op+' operation failed: Selected feature "'+selFeatureTitle+'" not found.')
+            logging.warning(op+' operation failed: Selected feature "'+selFeatureTitle+'" has an issue.')
```

Comment:
kind of makes sense - getFeature returns false even if there are multiple matches, so the message here should be vague, but again it's a duplicate message since sartopo_python logs a warning in both cases, so best is probably to pass or to get rid of the 'if not selFeature' clause entirely